### PR TITLE
Fix image_commands and user_packages

### PR DESCRIPTION
**Pull request by osallou**
Hi,
here are a few patches regarding image_commands and user_packages.

image_commands: various fixes to match updates on the _base_ (volume, filesystem,...). It was not working anymore.

user_packages: fix the issue #119, installing the packages at system modification instead of debootstrap due to described issue.
I also changed apt.py to use the debootstrap manifest mirror (or the default one). Indeed, it was set to a default debian mirror site, and may differ from bootstrap selected mirror. For user_packages download, it leaded to long downloads in my case versus my selected mirror (and different apt keys), or for final user it may not match the expected default mirror from the image creator.

Regarding user_packages I think you are rewriting it in python-WIP so you may need to adapt modifications in your branch.

### CODE REVIEW

File: plugins/image_commands/image_commands.py [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
         from common.tools import log_check_call
-
         for user_cmd in info.manifest.plugins['image_commands']['commands']:
             command = []
             for elt in user_cmd:
-                fragment = elt.format(
-                                root=info.root,
-                                image=info.loopback_file,
-                                filesystem=info.manifest.volume['filesystem'])
+                fragment = elt.format(root=info.root)
```
Comment (andsens) [inline plugins/image_commands/image_commands.py:17]: Why?

**Comment by andsens** [comment]
You should split this PR up. Some of the changes are bugfixes while others change behavior. Normally you'd have one fix/enhancement per PR. This one is an _all or nothing_.

**Comment by andsens** [comment]
Also. Try verifying that you conform to most PEP8 guidelines by running `find . -name '*.py' | /usr/bin/grep -v minify_json | xargs pep8 --ignore=E101,E221,E241,E501,W191`. The WIP branch conforms fully to that spec.

File: plugins/image_commands/image_commands.py (hunk 1 above)
Comment (osallou) [inline plugins/image_commands/image_commands.py:17]: Le 14 déc. 2013 19:28, "Anders Ingemann" notifications@github.com a
écrit :

**Pull request closed without merging**